scripts/setup/requirements.build.txt

```diff
@@ -9,6 +9,4 @@
 click
 click-option-group
 
-# scripts/py_matter_idl/matter/idl
-jinja2
-lark
+-e ${PW_PROJECT_ROOT}/scripts/py_matter_idl
```

Comment:
Does this automatically update if we change source code?

We worked towards explicitly preventing (and making it extra hard) for people to install our python modules as part of the build system because when we allowed that we got very hard to debug cases where "installed python files" were old and causing issues even when code was updated. Does not happen in CI, but does happen in regular development when people re-bootstrap infrequently.

The goal is that if I pull a new code version, I will not get errors in case new matter-idl or similar things change.

Reply:
Yes, this installs package in the editable mode, so the only thing that is installed in the env is the "symlink" to the actual location of the package source code. It works like adding source/py_matter_idl to PYTHONPATH but in a way that the package is actually imported from default python location.

I will have to explicitly check how it will behave when we install another package in the matter namespace (I just remembered that I forgot to do it).